docs/data-output.md

```diff
@@ -0,0 +1,404 @@
+
+### Sunday Cycle
+
+A liturgical year consists of cycles that determines which portions of scripture are to be read.
+The Sunday cycle is used mainly for Sunday readings and some solemnity.
+
+- `Year A` denoted by the key `YEAR_A`
+- `Year B` denoted by the key `YEAR_B`
+- `Year C` denoted by the key `YEAR_C`
+
+### Weekday Cycle
+
+A liturgical year consists of cycles that determines which portions of scripture are to be read.
+The weekday cycle is used mainly for weekday readings.
+
+- `Year 1` or `Odd year` denoted by the key `YEAR_1`
+- `Year 2` or `Even year` denoted by the key `YEAR_2`
+
+### Psalter Weeks
+
+Inside every liturgical year, another psalter week cycle will determine which psalms and prayer to follow for the liturgy of hours.
+
+With the exception of the Easter Octave, each week in the liturgical year is assigned readings from the [Psalter](https://en.wikipedia.org/wiki/Roman_Breviary#The_Psalter).
+The psalter week cycle is composed of a repeated sequence of 4 four weeks. It restarts at the beginning of each season.
+There are also some rules that govern the set of Psalter readings used on particular occasions or seasons of the year.
+
+For convenient usage, romcal provides the psalter week number for all seasons:
+
+- `Week I` denoted by the key `WEEK_1`
+- `Week II` denoted by the key `WEEK_2`
+- `Week III` denoted by the key `WEEK_3`
+- `Week IV` denoted by the key `WEEK_4`
+
+---
+
+The cycle objects can be imported into consumer apps via:
+
+_ES6_
+
+```javascript
+import { LITURGICAL_CYCLES, SUNDAY_CYCLES, WEEKDAY_CYCLES, PSALTER_WEEKS } from 'romcal';
+```
+
+_CommonJS_
+
+```javascript
+var LITURGICAL_CYCLES = require('romcal').LITURGICAL_CYCLES;
+var SUNDAY_CYCLES = require('romcal').SUNDAY_CYCLES;
+var WEEKDAY_CYCLES = require('romcal').WEEKDAY_CYCLES;
+var PSALTER_WEEKS = require('romcal').PSALTER_WEEKS;
+```
+
+## Liturgical Colors
+
+[Liturgical colors are those specific colors used for vestments and hangings within the context of Christian liturgy. The symbolism of violet, white, green, red, gold, black, rose and other colors may serve to underline moods appropriate to a season of the liturgical year or may highlight a special occasion.](https://en.wikipedia.org/wiki/Liturgical_colours)
+
+romcal defines various liturgical colors which are:
+
+- `BLACK`
+- `GOLD`
+- `GREEN`
+- `PURPLE`
+- `RED`
+- `ROSE`
+- `WHITE`
+
+More information on how these colors are used for liturgical day can be found [here](https://en.wikipedia.org/wiki/Liturgical_colours#Roman_Catholic_Church)
+
+Liturgical colors can be read via the `dates[idx].data.meta.liturgicalColor` property in each date element in the array that `calendarFor` returns.
+
+The `LITURGICAL_COLORS` object can be imported into consumer apps via:
+
+_ES6_
+
+```javascript
+import { LITURGICAL_COLORS } from 'romcal';
+```
+
+_CommonJS_
+
+```javascript
+var LITURGICAL_COLORS = require('romcal').LITURGICAL_COLORS;
+```
+
+## Calendar sources [deprecated]
```

Comment:
This section is still there because I did not have time to check if it could be completely deleted. It was used internally by romcal to compute the precedence and calendar rules. But I've completely refactored this feature last winter, to make it more powerful and effective, and allows a future calendar inheritance feature.
It reappeared when the conversion of the codebase to TypeScript was initiated by pejulian. Maybe he hadn't seen that this feature had been completely refactored.

Reply:
If so, I think it should be done before v2 release, or even better: in this very PR.